website/src/components/quickstartGuideList/index.js

```diff
@@ -1,168 +1,295 @@
-  };
+  }, [selectedFilters, quickstartData]);
+
+  // Function to organize guides by section
+  const organizedGuides = useMemo(() => {
+    // Check if any filters are actually selected (not just initialized)
+    const hasActiveFilters = Object.values(selectedFilters)
+      .some(selected => selected && selected.length > 0);
+
+    if (hasActiveFilters) {
+      return {
+        filtered: filteredData
+      };
+    }
+
+    // When no filters are active, use the original quickstartData instead of filteredData
+    return CONFIG?.categories?.reduce((acc, category) => {
+      return {
+        ...acc,
+        [normalizeTitle(category.title)]: quickstartData.filter(guide => 
+          category.guides?.includes(guide.data.id)
+        )
+      };
+    }, {}) || {};
+  }, [filteredData, selectedFilters, quickstartData]);
+
+  // Add this useEffect to load favorites
+  useEffect(() => {
+    const favoriteIds = JSON.parse(localStorage.getItem('favoriteGuides') || '[]');
+    const favoriteGuides = quickstartData.filter(guide => 
+      favoriteIds.includes(guide.data.id)
+    );
+    setFavorites(favoriteGuides);
+  }, [quickstartData]);
 
   return (
     <Layout>
       <Head>
         <title>{metaTitle}</title>
         <meta property="og:title" content={metaTitle} />
-        <meta property="og:description" content={quickstartDescription} />
+        <meta 
+          property="og:description" 
+          content={description} 
+        />
       </Head>
       <Hero
-        heading={quickstartTitle}
-        subheading={quickstartDescription}
+        heading={title}
+        subheading={description}
         showGraphic={false}
         customStyles={{ marginBottom: 0 }}
         classNames={styles.quickstartHero}
-        callToActions={heroCTAs.map(guide => ({
-          title: guide.title,
-          href: guide.href,
-          onClick: () => handleCallToActionClick(guide.value),
-          newTab: guide.newTab
-        }))}
-        callToActionsTitle={'Popular guides'}
+        lightBackground={true}
       />
-      <section id='quickstart-card-section'>
-        <div className={`container ${styles.quickstartFilterContainer} `}>
-          {tagOptions && tagOptions.length > 0 && (
-            <SelectDropdown options={tagOptions} onChange={setSelectedTags} value={selectedTags} isMulti placeHolder={'Filter by topic'} />
-          )}
-          {levelOptions && levelOptions.length > 0 && (
-            <SelectDropdown options={levelOptions} onChange={setSelectedLevel} value={selectedLevel} isMulti placeHolder={'Filter by level'} />
-          )}
-          <SearchInput onChange={(value) => setSearchInput(value)} placeholder='Search Guides' />
+      <section id='quickstart-card-section' className={styles.quickstartCardSection}>
+        <div className={`container ${styles.quickstartFilterContainer}`}>
+          <div className={styles.filterHeader} onClick={() => setIsFilterExpanded(!isFilterExpanded)}>
```

Comment:
For the filter on mobile - it has extra space at the top to allow room for the nav. However, once you start scrolling and the nav disappears, the filter is fixed with a good amount of space above it. Would it be possible to have the filter more towards the top when the nav isn't present?

I'm cool with letting that slide if it ends up being too much work to try and make happen. But. feels a bit off having that much space.

![image](https://github.com/user-attachments/assets/529f6895-d172-407e-a8ec-6355b8f959c6)

Reply:
Just pushed up a fix for this in the latest build!